cadc-registry/src/main/java/ca/nrc/cadc/reg/client/LocalAuthority.java

```diff
@@ -96,7 +96,11 @@ public class LocalAuthority {
     private final Map<URI, Set<URI>> authorityMap = new TreeMap<>();
 
     public LocalAuthority() {
-        PropertiesReader propReader = new PropertiesReader(RegistryClient.CONFIG_FILE);
+        this(RegistryClient.DEFAULT_CONFIG_FILE_NAME);
+    }
+    
+    public LocalAuthority(final String configFileName) {
```

Comment:
For testing support, package access should be used. We don't want alternate config file names/locations to be part of the exposed API... and a comment like `// unit tests only`

If this _test ctor_ isn't really needed, then remove. If it is, a ctor with a `File` argument is much more explicit and allows overriding the name and location.

Reply:
Fixed